apps/worker/src/app/workflow/usecases/execute-bridge-job/execute-bridge-job.usecase.ts

```diff
@@ -231,7 +240,7 @@ export class ExecuteBridgeJob {
                 statusCode: response.statusCode,
                 retryCount: response.retryCount,
                 message: response.statusMessage,
-                ...(body && body?.length > 0 ? { raw: JSON.parse(body) } : {}),
+                ...(body && body?.length > 0 ? { raw: rawMessage } : {}),
```

Comment:
This was another case where we weren't safely handling the parsing of the response body, which can lead to no activity feeds being logged at all due to unhandled `ParseError`